crypto/elliptic-curve.unit-test.ts

```diff
@@ -164,3 +165,73 @@ function curveWithFields(params: CurveParams) {
   // return Curve, Field and Scalar
   return [Projective, Affine.Field, Affine.Scalar] as const;
 }
+
+// Twisted Edwards curve tests
+
+const Ed25519 = createCurveTwisted(TwistedCurveParams.Ed25519);
```

Comment:
What's your preferred convention? CurveTwisted or TwistedCurve? Asking because in affine I believe I have read both.